response.md

```diff
@@ -63,6 +63,10 @@ Additionally, [/reverse](reverse.md) queries will have a `distance` parameter, w
 
 ## Notable features
 
+## `coordinates`
+
+At present, all results returned from Mapzen Search are points. Following the [GeoJSON specification](http://geojson.org/geojson-spec.html#positions), these coordinates are in **longitude, latitude** order.
```

Comment:
Yeah, I'm actually wondering if we should mention that we plan to return polygons someday. It may be useful to tell people but not appropriate for documentation. The docs should probably not become more complicated to tell people how things _might_ be some day, they should state how things are now.